packages/tools/src/dconf.coffee.md

```diff
@@ -44,13 +44,17 @@ Run the command "dconf-editor" to navigate the database with a UI.
         continue if typeof v is 'string'
         config.properties[k] = v.toString()
       # Execute
-      await @execute (
-        command: """
-        dconf read #{key} | grep -x "#{value}" && exit 3
-        dconf write #{key} "#{value}"
-        """
-        code_skipped: 3
-      ) for key, value of config.properties
+      finalStatus = false
+      for key, value of config.properties
+        {status} = await @execute
+          command: """
+          dconf read #{key} | grep -x "#{value}" && exit 3
+          dconf write #{key} "#{value}"
+          """
+          code_skipped: 3
+        finalStatus = status if status
+      status: finalStatus
+        
```

Comment:
Is there a test illustrating this issue ? why not simply returning null instead of create status from variables. This way, the status is the one of the child actions.

Reply:
@wdavidw There were no proper tests for dconf. I will create one to illustrate this.

To my mind we return `false` status only when all the properties were not changed. And `true` if at least one is changed.

Reply:
If there are no test, then it is time to create one. Make it simple and remove those status and finalStatus variables, there are of no use.

Reply:
done